CHANGES.md

```diff
@@ -1,3 +1,23 @@
+Release 4.39 (release date: 2019-05-29)
+=======================================
+New Features
+------------------
+- Compression level is now configurable by setting the `gzip_compression_level`
+  option under `[GSUtil]`. The default value is `9`.
+
+Bug Fixes
+------------------
+- Fixed bug where IAM operations failed on buckets whose names lacked any
+  letters.
+
+Other Changes
+------------------
+- Python 3 compatibility.
+- Continuous integration reimplemented and improved.
```

Comment:
The only reason this would be relevant to external devs is that the contribution guidelines have changed -- the dev docs now say to submit a GitHub PR (instead of using Rietveld), which will trigger CI tests.  Maybe revise this to:

"Revised contribution guidelines to direct developers to submit GitHub pull requests instead of using the Rietveld code review tool."